source/MaterialXGenArnold/libraries/pbrlib/genosl/arnold/mx_generalized_schlick_brdf.osl

```diff
@@ -0,0 +1,26 @@
+#include "pbrlib/genosl/lib/mx_microfacet_specular.osl"
+
+void mx_generalized_schlick_brdf(float weight, color color0, color color90, float exponent, vector2 roughness, normal N, vector U, string distribution, BSDF base, output BSDF result)
+{
+    if (weight < M_FLOAT_EPS)
+    {
+        result = base;
+        return;
+    }
+
+    float NdotV = fabs(dot(N,-I));
+    color F = mx_fresnel_schlick(NdotV, color0, color90, exponent);
+    float avgRoughness = mx_average_roughness(roughness);
+
+    // Calculate directional albedo since we need
+    // to attenuate the base layer according to this.
+    color dirAlbedo = mx_ggx_directional_albedo(NdotV, avgRoughness, color0, color90);
+    float avgDirAlbedo = dot(dirAlbedo, color(1.0 / 3.0));
+
+    // NOTE: No compensation for multiscatter is done here since this is already
+    // handled internally in Arnold's microfacet closure.
```

Comment:
Override implementation for Arnold since energy compensation is none inside the closure.